mods/cef-titlebar-enabler-universal.wh.cpp

```diff
@@ -1910,7 +1920,7 @@ int InjectCTEV8Handler(cef_v8value_t* const* arguments, cef_v8value_t** retval)
         AddValueToObj(initialConfigObj, u"forceextensions", cef_v8value_create_bool(cte_settings.forceextensions));
         AddValueToObj(initialConfigObj, u"allowuntested", cef_v8value_create_bool(cte_settings.allowuntested));
         AddValueToObj(retobj, u"initialOptions", initialConfigObj);
-        AddValueToObj(retobj, u"version", u"0.7");
+        AddValueToObj(retobj, u"version", u"0.8");
```

Comment:
FYI you have [the `WH_MOD_VERSION` constant](https://github.com/ramensoftware/windhawk/wiki/Creating-a-new-mod#constants).